package.json

```diff
@@ -25,6 +25,7 @@
     "chartjs-adapter-date-fns": "^3.0.0",
     "chartjs-plugin-datalabels": "^2.2.0",
     "chrono-node": "^2.7.3",
+    "classnames": "^2.5.1",
```

Comment:
@eugeneoshepkov  Sure, I could use `clsx` instead of classnames. But I miss the point of combining `clsx` and `tailwind-merge`. Could you explain why we need this conjunction?